hack/quickstart/quickstart-gce.md

```diff
@@ -8,6 +8,28 @@ This can be changed to identify separate clusters.
 export CLUSTER_PREFIX=quickstart
 ```
 
+### Add your SSH public key to project under user core
+
+Quickstart scripts assume you can ssh into the `core` user. Either follow the commands below or go to the web console to add a key for the user `core`.
+
+**WARNING** The following process will clobber existing keys in your project's ssh metadata.
```

Comment:
You can do it per-instance, but they list it as deprecated. SGTM, will close in favor of original PR.